app/code/Magento/Catalog/Model/Product.php

```diff
@@ -668,7 +668,7 @@ public function getTypeId()
      */
     public function getStatus()
```

Comment:
If we're fixing this, let's set a proper return type here too.
```suggestion
    public function getStatus(): int
```